v2/arangodb/collection_documents_replace.go

```diff
@@ -90,6 +90,11 @@ type CollectionDocumentReplaceOptions struct {
 
 	// RefillIndexCaches if set to true then refills the in-memory index caches.
 	RefillIndexCaches *bool
+
+	// IsRestore is used to make insert functions use the "isRestore=<value>" setting.
+	// Note: This option is intended for internal (replication) use.
+	// It is NOT intended to be used by normal client. Use on your own risk!
+	IsRestore *bool
```

Comment:
I do not see isRestore param in the doc
https://www.arangodb.com/docs/3.12/http/document.html#create-a-document

Reply:
They are hidden from docs on purpose